dbl/client.py

```diff
@@ -186,7 +187,7 @@ def guild_count(self):
         ==========
 
         limit: int[Optional]
-            The number of results you wish to lookup. Defaults to 50.
+            The number of results you wish to lookup. Defaults to 50. Max 500.
         offset: int[Optional]
             The page number to search. Defaults to 0.
```

Comment:
Could you change the wording of this? It's a bit misleading as it's the amount of bots to skip. To get to the next "page", you'd want to skip `limit * (page - 1)` bots.

Reply:
Is this good enough?

Reply:
Yes